types/locale/index.d.ts

```diff
@@ -0,0 +1,106 @@
+// Type definitions for locale 0.1
+// Project: https://github.com/florrain/locale
+// Definitions by: Emily Marigold Klassen <https://github.com/forivall>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+export = locale;
+
+declare global {
+    namespace Express {
+        interface Request {
```

Comment:
Sorry to be obtuse, but I'm not sure what this is for.  Could you maybe add a sample usage in the test file?

Reply:
The test file has a usage at https://github.com/DefinitelyTyped/DefinitelyTyped/blob/8ec0ab5eb0283bb7ef715ec3b4b68f11e1f93cd0/types/locale/locale-tests.ts#L44 - See https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/express-serve-static-core/index.d.ts#L19 for info. Most declarations that use this functionality don't have anything to describe what this global augmentation does, but `passport` does have the line `// These declarations are merged into express's Request type`

Reply:
Thanks for the explanation!